test/suites/para/test_tanssi_containers.ts

```diff
@@ -177,5 +180,169 @@ describeSuite({
+        let alice = keyring.addFromUri('//Alice', { name: 'Alice default' });
+
+        // Read raw chain spec file
+        // Different path in CI: ./specs vs ../specs
+        let spec2002 = null;
+        try {
+            spec2002 = await fs.readFile("./specs/template-container-2002.json", 'utf8');
+        } catch {
+            spec2002 = await fs.readFile("../specs/template-container-2002.json", 'utf8');
+        }
+
+        // Augment paraApi with new RPC method
+        // TODO: latest moonwall version supports this in beforeAll
+        const wsProvider2 = new WsProvider('ws://127.0.0.1:9948');
+        let paraApi2 = await ApiPromise.create({ provider: wsProvider2,
+          types: {
+            ContainerChainGenesisData: {
+              storage: "Vec<ContainerChainGenesisDataItem>",
+              name: "Vec<u8>",
+              id: "Vec<u8>",
+              fork_id: "Option<Vec<u8>>",
+              extensions: "Vec<u8>",
+              properties: "TokenMetadata",
+            },
+            TokenMetadata: {
+              token_symbol: "Vec<u8>",
+              ss58_format: "u32",
+              token_decimals: "u32",
+            },
+            ContainerChainGenesisDataItem: {
+              key: "Vec<u8>",
+              value: "Vec<u8>",
+            }
+          },
+          rpc: {
+            utils: {
+              raw_chain_spec_into_container_chain_genesis_data: {
+                description: 'Convert a raw chain spec string into a ContainerChainGenesisData',
+                params: [
+                  {
+                    name: 'raw_chain_spec',
+                    type: 'Text'
+                  }
+                ],
+                type: '(u32, ContainerChainGenesisData)'
+              }
+            }
+          }
+        });
+        let spec2002text = paraApi2.createType('Text', spec2002);
+        const containerChainGenesisDataFromRpc = await paraApi2.rpc.utils.raw_chain_spec_into_container_chain_genesis_data(spec2002text);
+        expect(containerChainGenesisDataFromRpc[0].toNumber()).to.be.equal(2002);
+
+        // Before registering container chain 2002, ensure that it has 0 blocks
+        // Since the RPC doesn't exist at this point, we need to get that from the relay
+        const header2002 = await getHeaderFromRelay(relayApi, 2002);
+        expect(header2002.number.toNumber()).to.be.equal(0);
+        const registered1 = (await paraApi.query.registrar.registeredParaIds());
+        expect(registered1.toJSON().includes(2002)).to.be.false;
+
+        const tx = paraApi.tx.registrar.register(2002, containerChainGenesisDataFromRpc[1]);
+        await signAndSendAndInclude(paraApi.tx.sudo.sudo(tx), alice);
+        const session1 = (await paraApi.query.session.currentIndex()).toNumber();
+        await jumpSessions(context, 2);
+        const session2 = (await paraApi.query.session.currentIndex()).toNumber();
+        console.log("Session before/after: ", session1, "/", session2);
```

Comment:
Now the tests are failing because in CI `await jumpSessions(context, 2);` only waits for 1 session. Locally it is working as expected (prints 1/3), not sure why CI prints (1/2).